src/modules/rabbitmq/RabbitMQModule.ts

```diff
@@ -44,13 +36,24 @@ import TimeoutException from '../../exceptions/TimeoutException';
         MetricsHandler,
     ],
 })
-export default class RabbitMQModule implements OnApplicationBootstrap {
-    private readonly logger = new Logger('Redis');
+export default class RabbitMQModule implements OnApplicationBootstrap, OnApplicationShutdown {
+    private readonly logger = new Logger(RabbitMQModule.name);
 
     constructor(
+        @Inject(TYPES.rabbitMqConnection) private readonly rabbit: AmqpConnectionManager,
         private readonly apiMqPublisher: ApiMQPublisher,
         private readonly apiMqDelayPublisher: ApiMQDelayPublisher,
     ) {
+      rabbit
+      .on('connect', () => {
+        this.logger.log('Connected');
+      })
+      .on('disconnect', () => {
+        this.logger.log('Disconnected');
+      })
+      .on('connectFailed', () => {
+        this.logger.warn('Connection failed');
```

Comment:
I'd throw an `ApplicationException` here so it kills the app, without rabbit it won't work properly anyway

Reply:
Yeah, but then the application would go into a boot loop. Besides the health module should be the one that takes care of this problem. The kubernetes should give some sort of alert that the application is unhealthy.

Reply:
Also in the case that rabbitmq would have a small outage(due to update) it would crash the aggregator while it would have come back in a few seconds/minutes.

Reply:
Fyi app currently isn't in k8s, but yes I understand the point, it should attempt to reconnect and issue a warning instead.